libs/island-ui/contentful/src/lib/TellUsAStoryForm/TellUsAStoryForm.tsx

```diff
@@ -18,15 +18,14 @@ import {
   ToastContainer,
   toast,
 } from '@island.is/island-ui/core'
-import BackgroundImage from '../BackgroundImage/BackgroundImage'
-import { Slice as SliceType, richText } from '../..'
 import { useWindowSize, useIsomorphicLayoutEffect } from 'react-use'
 import { theme } from '@island.is/island-ui/theme'
+import { Locale } from '@island.is/shared/types'
 
+import BackgroundImage from '../BackgroundImage/BackgroundImage'
+import { Slice as SliceType, richText } from '../..'
 import * as styles from './TellUsAStoryFrom.treat'
 
-type Locale = 'is' | 'pl'
```

Comment:
`pl` out?

Reply:
Yes, afaik, Polish it's not configured yet inside Contentful so this locale doesn't do anything. It would require more work to make it work in the whole project, so I removed it until a bigger task is done to support it everywhere.